.rhiza/tests/integration/test_notebook_execution.py

```
@@ -66,7 +74,9 @@ def test_notebook_execution(notebook_path: Path):
         "/dev/null",  # We don't need the actual HTML output
     ]
 
-    result = subprocess.run(cmd, capture_output=True, text=True, cwd=notebook_path.parent)
+    result = subprocess.run(
```

Ruff (0.14.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)